ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/how-to/custom-templates.md

```diff
@@ -0,0 +1,47 @@
+<!--
+  ~ Copyright (c) 2023 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
+
+# Custom Templates
+
+With Custom Templates for the `eos_cli_config_gen` role, the user can create additional CLI configuration for features that are not covered by the built-in templates.
+
+## Create a Custom Template
+
+Custom templates should be written in Jinja2.
+
+For details on how to create Jinja2 templates, see [Templating Jinja2](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_templating.html#templating-jinja2)
+
+Custom templates can utilize any AVD variable for variable substitution including additional structured configuration variables created by a custom template in the `eos_designs` role. For details on how to create additional structured configuration variables see [Custom Templates for eos_designs](../../../eos_designs/docs/how-to/custom-templates.md) 
+
```

Comment:
Maybe add similar details here on where the file should be placed, for Jinja2 to be able to pick it up. (Hint: It only works below the playbook dir, so you might have to use symlinks if it is placed elsewhere)

Reply:
Amended in the latest push.